.github/workflows/remove-labels.yml

```diff
@@ -0,0 +1,17 @@
+name: Remove PR Labels
+
+on:
+  pull_request_target:
+    types: ["closed"]
```

Comment:
is there a difference between `closed` and `merged` or this will cover both cases?

Reply:
Thanks for pointing this out, Mai. I think we'll have to add `merged` to this list. 
We didn't need it for the Bazel repo because PRs are `closed` by the bot. @sgowroji

Reply:
And yeah, the `awaiting-review` label will be removed from all closed(/merged) PRs moving forward (not old ones).

Reply:
Sure. Thanks for suggestion.